packages/frontend/web-ui/src/game/models/CreateNewGameHttpRequest.ts

```diff
@@ -0,0 +1,7 @@
+import { GameOptions } from './GameOptions';
+
+export interface CreateNewGameHttpRequest {
```

Comment:
Let's rely on existing API models

Reply:
Changed to the API models

Reply:
rely on `apiModels.GameCreateQueryV1`.

Reply:
I've removed this interface because I didn't use anymore.